src/ducks/projects.js

```diff
@@ -32,7 +32,8 @@ const fetchProjects = () => {
       type: FETCH_PROJECTS,
     });
     const query = {
-      current_user_roles: ['owner', 'translator']
+      current_user_roles: ['owner', 'translator'],
+      include: ['workflows']
```

Comment:
I'm thinking just don't include them here. They aren't needed until you select a project to work on.

Projects like NfN and Wildcam Gorongosa, with large numbers of workflows, would still be a problem.